### ISSUE TITLE
Create issue forms

### ISSUE THREAD

**Issue by Mister-X-** [aircrack-ng/aircrack-ng#2356]
# Description

To simplify bug reporting, we can now use [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) which allows to have fields to split the information.

**Comment by gemesa** [comment]
What should be done here exactly? [bug_report.md](https://github.com/aircrack-ng/aircrack-ng/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) and [feature_request.md](https://github.com/aircrack-ng/aircrack-ng/blob/master/.github/ISSUE_TEMPLATE/feature_request.md) are already available, do we need some other form?

**Comment by Mister-X-** [comment]
So, now GitHub allows to have issue forms, like every other bug tracking tool, as opposed to free form that we've had since forever on GitHub, where you can write whatever you want.

While free form works for the majority of people, sometimes we have to spend time asking for information because people have erased the whole thing and decided to write it however they want, without providing the needed information to make the bug report actionable.

**Comment by Mister-X-** [comment]
Yes. Github now offer to be able to create bug report forms like every other bug tracking tool.

So, I need to convert the issue report free form into an issue form, to force some of the field to be filled with information, and I hope that's going to decrease the amount of unactionable bug reports
